Backend/mentii/user_ctrl.py

```diff
@@ -172,16 +174,81 @@ def getUserByEmail(email, dbInstance):
 
   table = dbUtils.getTable('users', dbInstance)
   if table is None:
-    MentiiLogging.getLogger().error("Unable to get table users in getUserByEmail")
+    MentiiLogging.getLogger().error('Unable to get table users in getUserByEmail')
     return None
 
-  key = {"Key" : {"email": email}}
+  key = {'Key' : {'email': email}}
   result = dbUtils.getItem(key, table)
   if result is None:
-    MentiiLogging.getLogger().error("Unable to get the user with email: " + email + " in getUserByEmail ")
+    MentiiLogging.getLogger().error('Unable to get the user with email: ' + email + ' in getUserByEmail ')
     return None
```

Comment:
Refactor later to remove return statement. It's explicit, but not necessary to return at this point